R/oauth-flow-auth-code.R

```diff
@@ -123,20 +125,27 @@ req_oauth_auth_code <- function(req, client,
 #' @param auth_params List containing additional parameters passed to `oauth_flow_auth_code_url()`
 #' @param token_params List containing additional parameters passed to the
 #'   `token_url`.
-#' @param host_name `r lifecycle::badge("deprecated")` Use `redirect_uri`
-#'   instead.
+#' @param host_name,host_ip,port `r lifecycle::badge("deprecated")`
+#'   Now use `redirect_uri` instead.
 #' @param host_ip IP address for the temporary webserver used to capture the
 #'   authorization code.
-#' @param type Either `desktop` or `web`. Use desktop when running on the
-#'   desktop in an environment where you can redirect the user to `localhost`.
-#'   Use `web` when running in a hosted web environment.
-#' @param port Port to bind the temporary webserver to. Used only when
-#'   `redirect_uri` is `"http(s)://localhost"`. By default, this uses a random
-#'   port. You may need to set it to a fixed port if the API requires that the
-#'   `redirect_uri` specified in the client exactly matches the `redirect_uri`
-#'   generated by this function.
+#' @param port Port to bind the temporary webserver to.
```

Comment:
Shouldn't the explicit docs for `host_ip` and `port` go away, since they are lumped in with the `host_name` as deprecated in favor of `redirect_uri`?